qiskit/transpiler/passes/__init__.py

```diff
@@ -227,8 +230,11 @@
 
 # circuit scheduling
 from .scheduling import TimeUnitConversion
+from .scheduling import ALAPScheduleAnalysis
+from .scheduling import ASAPScheduleAnalysis
 from .scheduling import ALAPSchedule
 from .scheduling import ASAPSchedule
+from .scheduling import DynamicalDecouplingPadding
```

Comment:
Then the name should be `PadDynamicalDecoupling` or rename `PadDelay` to `DelayPadding`? I think both should be okey. Anyways I am not a heavy user of this pass, so I'd like to hear @ajavadia 's thought. 

> because anyone that was using DynamicalDecoupling today successfully 
No one can use the pass successfully today. Anyways I agree if someone has custom DD pass  this change will break their code.

Reply:
I'm almost good with this PR. Just holding my approval until we hear some user's voice about the class name and future deprecation plan.

Reply:
yeah I think `ASAPScheduleAnalysis` and `ALAPScheduleAnalysis` and `PadDynamicalDecoupling` are fine, they are explicit about what they do.